eng/common-variables.yml

```diff
@@ -2,16 +2,12 @@
 variables:
```

Comment:
Do we really need this common-variables.yml?  What not put it inline in diagnostics.yml? Especially since most of the variables have been removed.

Reply:
Deleted and cleaned.